HW_1/task1.py

```diff
@@ -0,0 +1,16 @@
+#Задача 2: Найдите сумму цифр трехзначного числа.
+
+#*Пример:*
+
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0) 
+
+#n = input("Введите трехзначное число: ")
+#n = int(n)
+
+#d1 = n % 10
+#n = n // 10
+#d2 = n % 10
+#n = n // 10
```

Comment:
Можно сократить на одну строку.
```suggestion
#d1 = n % 10
#d2 = n // 10 % 10
#n = n // 100
```